### PR TITLE
Feature/7428 amend launch script to do dependency update

### PR DESCRIPTION
**Pull request by gavinwakeley1**
AB#7428

### CODE REVIEW

File: launch-or-update-local.ps1 [hunk 1]
```diff
@@ -1 +1,25 @@
...
+### usage:                                       ###
+###  ./launch-or-update-local.ps1                ###
+### or without downloading updates...            ###
+###  ./launch-or-update-local.ps1 -update false  ###
+####################################################
+
+param(
+        [Parameter()]
+        [ValidateSet('','true','false')]
+        [string]$update
```
Comment (talawson05) [inline launch-or-update-local.ps1:12]: so in shell i can use either -u or -update; but in powershell it's only -update?

Comment (gavinwakeley1) [reply launch-or-update-local.ps1:12]: Good point Tony, I'll add an additional attribute for -u to Powershell also

Comment (talawson05) [reply launch-or-update-local.ps1:12]: it looks like it's -u and --update on shell. is that right? Can it be the same on powershell?

Comment (gavinwakeley1) [reply launch-or-update-local.ps1]: Ah ok, I get what you mean now, I'll sort that

**Pull request merged**